templates/download.html.twig

```diff
@@ -41,6 +41,16 @@ php -r "unlink('composer-setup.php');"</pre>
         <li>Remove the installer</li>
     </ul>
 
+    <p>Most likely, you want to put the <code>composer.phar</code> into a directory on your PATH, so you can simply call <code>composer</code> from any directory (<em>Global install</em>):
+    </p>
+    <pre class="language-bash">sudo mv composer.phar /usr/local/bin/composer</pre>
+    <p>
+        For details,
+        <a href="/doc/00-intro.md#globally"
+           title="See the instructions on how to install Composer globally"
+           aria-label="See the instructions on how to install Composer globally">see the instructions on how to install Composer globally</a>.
```

Comment:
remove the `aria-label` entirely. It is useless now as it duplicates the text of the link